### PR TITLE
firmware: ipq-wifi: update to version 2024-07-04

### PR DESCRIPTION
**Pull request by skramstad**
**Contains following updates:**
[+] qcn9074: add Cambium Networks XE3-4 BDF
[+] ipq60xx: add Cambium Networks XE3-4 BDF

Signed-off-by: Kristian Skramstad <kristian+github@83.no>

### CODE REVIEW

**Comment by skramstad** [comment]
@robimarko I ran `make package/check FIXUP=1` but no fixes was present. 
So I don't really know why some checks were not successful.

File: package/firmware/ipq-wifi/Makefile [hunk 1]
```diff
@@ -2,13 +2,13 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/version.mk
 
 PKG_NAME:=ipq-wifi
-PKG_RELEASE:=1
+PKG_RELEASE:=2
```
Comment (hauke) [inline package/firmware/ipq-wifi/Makefile:5]: Please keep the `PKG_RELEASE` at 1. This is increased when something changes in the package without changing the version, but you changed `PKG_SOURCE_DATE` so you should reset it to 1.

Comment (skramstad) [reply package/firmware/ipq-wifi/Makefile]: Ah okey, sorry about that. Pushed new commit.

**Comment by skramstad** [comment]
@hauke Do you know why the build failed? I did run `make package/check FIXUP=1` as recommended, but did not see any issues/errors.

**Comment by skramstad** [comment]
@robimarko any idea why this build failed? As mentioned earlier I did check for fixup, but none present.

**Comment by robimarko** [comment]
@skramstad The hash is incorrect, I just checked locally.

Do you have OpenWrt tools built?
Cause if not then your system ZSTD is used instead and that is where the hash difference is

**Comment by skramstad** [comment]
> @skramstad The hash is incorrect, I just checked locally.
> 
> Do you have OpenWrt tools built? Cause if not then your system ZSTD is used instead and that is where the hash difference is

Hmm... I believe I don't have OpenWrt tools built. What is the command to build that tool?

**Comment by robimarko** [comment]
That would be the issue then.

`make tools/zstd/compile` should be enough to be able to calculate the right hash

**Comment by skramstad** [comment]
Great. I'll do that and commit when done.

**Comment by skramstad** [comment]
Yeah, so this was my mistake. I didn't pay attention to my ipq-wifi file. I previously added xe34 device, so that would be a an issue. I fixed it, and did a new fixup.

Looks better now. Thanks for the hint @robimarko 👍

**Comment by robimarko** [comment]
Thanks! Rebased on top of main and merged!

**Pull request merged**